CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## <sub>v2.1.0-alpha.0</sub>
```

Comment:
Does the release tooling manage re-titling when we flip over to non-alpha?

Reply:
No, have to handle that manually. Might almost be better to not do the titling for alpha releases.